buildlib/pr/codestyle.yml

```diff
@@ -70,3 +70,25 @@ jobs:
               azure_complete_with_issues "Code is not formatted according to the code style, see $url for more info."
           fi
         condition: eq(variables['Build.Reason'], 'PullRequest')
+
+  - job: codespell
+    displayName: codespell check
+    pool:
+      name: MLNX
+      demands:
+      - ucx_docker -equals yes
+    container: fedora
+    steps:
+      - checkout: self
+        clean: true
+        fetchDepth: 100
+        retryCountOnTaskFailure: 5
+
+      - bash: |
+          python3 -m venv /tmp/codespell_env
+          source /tmp/codespell_env/bin/activate
+          pip3 install codespell
+          codespell
+          test_exit_code=$?
```

Comment:
Do we really need to store an exit code?
If `codespell` command fails, Azure will fail the job.

Reply:
That was needed because of env cleanup, now it should work without saving exit code